pages/api/verify.ts

```diff
@@ -34,6 +36,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse<Data>) => {
     where: {
       discordServerId: body.discordServerId,
       discordMemberId: body.discordMemberId,
+      starknetNetwork: body.network,
     },
     relations: ["discordServer"],
```

Comment:
sometimes you use `discordServer` in relations and sometimes `DiscordServer` . Only one of them is correct I guess, can you double check?